Orange/widgets/tests/base.py

```diff
@@ -215,31 +221,42 @@ def send_signal(self, input, value, *args, widget=None, wait=-1):
             else:
                 raise ValueError("'{}' is not an input name for widget {}"
                                  .format(input, type(widget).__name__))
+        if widget.isBlocking():
```

Comment:
Of the widget is blocking the code will not get past this check and therefore the "wait" parameter is not used. Was this intended?